app/src/main/java/com/vultisig/wallet/ui/screens/vault_settings/components/ConfirmDeleteScreen.kt

```diff
@@ -120,9 +120,7 @@ internal fun ConfirmDeleteScreen(
                         bottom = 16.dp,
                     ),
                 text = stringResource(R.string.confirm_delete_delete_vault),
-                onClick = ClickOnce {
-                    onConfirmClick()
-                },
```

Comment:
The reason for the change is that the MultiColorButton(its parent) handles ClickOnce.